crates/release_plz_core/src/command/update.rs

```diff
@@ -47,6 +48,15 @@ impl PackagesUpdate {
     }
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct ReleaseInfo {
```

Comment:
I'm not sure this name is right, I would love some suggestions if you have any

Reply:
looks good